spec/service_consumers/pact_helper.rb

```diff
@@ -204,4 +204,34 @@
       FactoryGirl.create(:version, target: arabic_draft, number: 1)
     end
   end
+
+  provider_state "the content item bed722e6-db68-43e5-9079-063f623335a7 is at version 3" do
+    set_up do
+      DatabaseCleaner.clean_with :truncation
+
+      draft = FactoryGirl.create(:draft_content_item, content_id: "bed722e6-db68-43e5-9079-063f623335a7")
+      FactoryGirl.create(:version, target: draft, number: 3)
+
+      stub_default_url_arbiter_responses
+      stub_request(:put, /content-store/)
```

Comment:
There's a lot of inconsistency in these different setups.  Some don't stub them at all but don't have any indication as to why.

What's the likelihood of this matching something unexpected?